packages/core/stories/card/card.stories.tsx

```diff
@@ -131,7 +131,7 @@ export const AccentVariations: StoryFn<typeof Card> = (args) => {
 };
 
 export const Variants: StoryFn<typeof Card> = (args) => {
-  const variants = ["primary", "secondary", "tertiary"] as const;
+  const variants = ["primary", "secondary", "tertiary", "alpha"] as const;
```

Comment:
Safety check.
Primary, secondary and tertiary are a scale, mutually exclusive and agnostic of the presentation style.
Where does alpha fit into the visual hierachy, as mentioned by the docs?
What if there was a use-case where you wanted alpha variants of the primary, secondary or tertiary? 
Alpha would then become a mis-nomer, so from an end-users POV, not that intuitive to use.
Should this be a boolean  and a separate prop ?

Reply:
There won't be clear guidance for combining alpha with p/s/t variants. There should only be one alpha within the system. "alpha" (or calling it something else), is correctly sitting on a scale of both `variant` (P>S>T>A) as well as Button's `appearance` (S>B>T>A)